modules/data_warehouse/main.tf

```diff
@@ -405,15 +333,17 @@ resource "google_project_service_identity" "bigquery_data_transfer_sa" {
   service  = "bigquerydatatransfer.googleapis.com"
 }
 
-resource "google_project_iam_member" "dts_permissions_token" {
-  project = data.google_project.project.project_id
-  role    = "roles/iam.serviceAccountTokenCreator"
-  member  = "serviceAccount:${google_project_service_identity.bigquery_data_transfer_sa.email}"
-}
+# # Grant the DTS service account access
+resource "google_project_iam_member" "dts_service_account_roles" {
+  for_each = toset([
+    "roles/iam.serviceAccountTokenCreator",
```

Comment:
Do we need token creator at the project level or can we scope this to a single project?